### PR TITLE
Przemek backend person in text

### PR DESCRIPTION
**Pull request by Isild**

### CODE REVIEW

File: scripts/algorithm.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    
+
+
+index += 1
+weight = 2**index
+#funkcja dzieląca tekst i ustalająca wagi
+#dependencySearch(text1.split('.'), dependendencyTable, weight, personsTable)
+print("Tablica zależności po szukaniu w oknie: ")
+for d in dependendencyTable:
+    print(d)
```
Comment (lpawelcz) [inline scripts/algorithm.py:223]: Is this file actually needed?

**Comment by lpawelcz** [comment]
Please write comments in english and delete unnecessary ones

**Comment by baatochan** [comment]
Not gonna lie, its hard to me to do a review when I completely dont know what Im reviewing. What is that? What are these commits all about?

File: scripts/algorithm.py (hunk 1 above)
Comment (wojtasiq) [reply scripts/algorithm.py:223]: @Isild ?

**Comment by lpawelcz** [comment]
imma clean this PR

**Comment by lpawelcz** [comment]
as for the conflict: `basic.py` is obsolete, it is safe to delete it

File: scripts/algorithm.py (hunk 1 above)
Comment (lpawelcz) [reply scripts/algorithm.py]: deleted unnecessary files

**Comment by lpawelcz** [comment]
@Isild merge plz

**Comment by baatochan** [comment]
WHY WAS IT MERGED WITHOUT ALL TESTS PASSED?!
